OrbitProducerSideChannel/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+project(OrbitProducerSideChannel)
+
+add_library(OrbitProducerSideChannel INTERFACE)
+target_compile_options(OrbitProducerSideChannel INTERFACE ${STRICT_COMPILE_FLAGS})
```

Comment:
You probably want to remove this line. This would enforce the strict compiler flags on all users of this module. And that's what we want to avoid be enabling it on a module-by-module-basis.

Reply:
Done.